app/views/books/_translations.html.erb

```diff
@@ -1,4 +1,8 @@
 <p>
+This book is available in
+  <a href="/book/en">English</a>.
+</p>
+<p>
```

Comment:
Yeah, I think that makes sense. The fact that "English" is by itself in the first block probably makes it clear that it is the primary language.